CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 ### Changed
 
 - Error out when socket fails to create with `--daemon`
+- Hints execution no longer sets its cwd to the shell's working directory
```

Comment:
Should word around spawning a program, since that's what is relevant in both cases. 
```suggestion
- Setting current working directory to non existent one when spawning a program
```

And in `Fixed` section.